### PR TITLE
add 2019 page

### PR DESCRIPTION
**Pull request by brute4s99**
added the 2019 page. This page still needs to be linked with the dgplug.org landing page

### CODE REVIEW

**Comment by kushaldas** [comment]
@brute4s99 Please rebase and also sign the commit.

**Comment by brute4s99** [comment]
Will do by the end of the day!

Best regards
Piyush Aggarwal | brute4s99
https://piyush.tech/

**Comment by brute4s99** [comment]
done, @kushaldas ! ^-^

File: content/summertraining19/contents.lr [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+IST (13:00 UTC) onwards.
+
+We will be spending time to learn about ethics in programming and in life. Even
+though we all get a chance to learn a lot of programming and technology
+regularly, but, we don't get to see how those technical decisions affect our
+freedom. We want to help people not only to learn about how to use a tool, or
+how to use a framework to develop the next best application, but, we also want
+people to think about the consequence of the code we write. The current situation
+in the world also raises questions on users' privacy and freedom both on Internet
+and on real life. We wil help the participants to learn the technologies and also
```
Comment (vivekanand1101) [inline content/summertraining19/contents.lr]: *will

**Comment by kushaldas** [comment]
@brute4s99 You will have to add your gpg key to github, it says `unverified` and can not be merged.

**Pull request merged**